packages/devtools-launchpad/src/components/LandingPage.js

```diff
@@ -65,49 +65,41 @@ const LandingPage = React.createClass({
     );
   },
 
-  renderFirefoxPanel() {
-    const targets = getTabsByClientType(this.props.tabs, "firefox");
-    return dom.div(
-      { className: "center" },
-      this.renderTabs("", targets, "firefox-tab"),
-      firstTimeMessage("Firefox", "firefox")
-    );
-  },
-
-  renderChromePanel() {
-    const targets = getTabsByClientType(this.props.tabs, "chrome");
-    return dom.div(
-      { className: "center" },
-      this.renderTabs("", targets, "chrome-tab"),
-      firstTimeMessage("Chrome", "chrome")
-    );
-  },
-
-  renderNodePanel() {
-    const targets = getTabsByClientType(this.props.tabs, "node");
-    return dom.div(
-      { className: "center" },
-      this.renderTabs("", targets, "node-tab"),
-      firstTimeMessage("Node", "node")
-    );
-  },
-
   renderPanel() {
-    const panels = {
-      Firefox: this.renderFirefoxPanel,
-      Chrome: this.renderChromePanel,
-      Node: this.renderNodePanel
+    const configMap = {
```

Comment:
@jasonLaster I forgot to do this. I'll put it in a PR with other some small improvements. 
What do you mean by module context here ? this file https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/index.js ?